app/components/form/step_navigation_component.html.erb

```diff
@@ -0,0 +1,34 @@
+<div>
```

Comment:
new component for step navigation via buttons that resemble carousel designs (chevron buttons)